packages/documentation/src/pages/forms/form-tutorial-basic.mdx

```diff
@@ -1,12 +1,13 @@
 ---
-title: VAFS tutorial - basic
+title: VAFS - basic tutorial
+tags: yeoman generator, new form, setup form,
```

Comment:
mostly tagging unique words. some of the pages don't have any unique words beyond their title